portfolio/src/main/webapp/index.html

```diff
@@ -8,49 +8,88 @@
-          </p>  
-        <p>I am not just my heritage, I am a young man with a goal to have others understand who I am and surround myself with people who will support me and challenge me. Education is what is going to help me achieve understanding, but I believe it is through the process that I will gain the relationships and knowledge that will lead to creating the network of people I strive to be a part of. That’s why I was so excited to go to college: because one of the fundamental functions of college is to bring a many of these kinds of people together. It gives me the ability to actualize my aspirations.
-          </p>
-      </p>
-      <h2>Interests</h2>
-      <h3>Cooking</h3>
-        <img src="images/Pizza.jpg" alt="Margherita Pizza" style="width:403px;height:302px;">
-        <p></p>
-        <img src="images/Me_Making_Pizza.jpg" alt="Making Pizza" style="width:302px;height:403px;">
-        <p>These two pictures come from a family pizza night. Made some leafy margherita pizza for my sisters and some sausage calzones for my parents!</p>
-      <h3>Art</h3>
-        <h4>Ilah</h4>
-        <img src="images/Ilah.jpg" alt="Ilah" style="width:302px;height:403px;">
-        <p></p>
-        <h4>Self Portrait</h4>
-        <img src="images/selfart.jpg" alt="Digital Render Self Portrait" style="width:302px;height:403px;">
-        <p></p>
-        <h4>Black Curtain</h4>
-        <img src="images/curtains.jpg" alt="Black Curtains" style="width:302px;height:403px;">
-        <p>These are some pieces that I've created in the past year. I really enjoy working with pencil, pen, and alcohol markers but I've also started to work with Photoshop and Procreate to try some digital art. The self portrait is the first piece is the first artwork I've created digitally.</p>
-      <h2>Experience</h2>/
-      <h2>Goals</h2>
-      <div class="footer">
-            <h2>Contact Me</h2>
-            <h3>You can ask me about anything on my page!</h3>
-            <p>Phone: +1 (323) 572-4214</p>\
-            <p>Google Email: amanhey@google.com</p>
-            <p>Oxy Email: amanhey@oxy.edu</p>
+
+      <h1 class="header">Adrian's Portfolio</h1>
+      <subtitle>Computer Science Student at Occidental College</subtitle>
+      
+      <div class="navigationbar">
+          <a href="#home_tab">Home</a>   
+          <a href="#interests_tab">Interests</a>
+          <a href="#experience_tab">Experience</a>
+          <a href="#goals_tab">Goals</a>
```

Comment:
Do you need to remove the experience and goals tab. It looks like they are not being used.

Reply:
Yeah I made some changes since this pull request that I tried to commit but couldn't create another pull request. Right now my working tree is clean. I deleted those two things for now to make things less cluttered, I'm planning to add them back once I have the functionality I want.

Reply:
Sounds good.